File: Iniciante/1005 - Média 1/bee1005.cpp
// Beecrowd 1005 - Média 1
#include <iomanip>
#include <iostream>
using namespace std;

int main() {

  double A, B;
  float media;

  cin >> A;
  cin >> B;

  media = ((A * 0.35) + (B * 0.75)) / 1.1;

  cout << "MEDIA = ";
  cout << fixed << setprecision(5) << media << "\n";

  return 0;

  return 0;
}

File: Iniciante/1005 - Média 1/bee1005.jl
# 1005 - Média 1

using Printf

function main()
  a = parse(Float64, readline())
  b = parse(Float64, readline())
  c = (a*3.5 + b*7.5) / 11.0
  @printf("MEDIA = %.5f\n", c)
end

main()
